src/domain/runtime/type/remote_object/type/object/property_preview.rs

```diff
@@ -71,6 +71,18 @@ impl PropertyPreview {
                         value.to_string()
                     }
                 }
+                Type::Accessor => {
+                    let r#type = if let Some(subtype) = &self.subtype {
+                        format!("ttt{:?}", subtype)
+                    } else {
+                        "Object".to_string()
+                    };
+                    let mut value = format!("[{}]", r#type);
+                    if cfg!(feature = "color") {
+                        value = format!("tpt{}", style(value).cyan())
```

Comment:
what is tpt?

also - what made you pick `cyan` here? it might be right but usually the way i pick the colors is by console.log(obj) in a node script and then copying the color from there

Reply:
I just picked a color but I'll go do that